README.md

```diff
@@ -1105,7 +1105,7 @@ You can configure yt-dlp by placing any supported command line option to a confi
     * `~/yt-dlp.conf`
     * `~/yt-dlp.conf.txt`
 
-    PS: See [Notes about environment variables](#notes-about-environment-variables)
+    P.S. See [Notes about environment variables](#notes-about-environment-variables)
```

Comment:
`PS:` is better in my opinion

Reply:
I would write `Check ... too.` for there

Reply:
This was meant to be a cleanup PR, not a make-a-mess PR 🤦‍♂️

Reply:
`See also: ` ?